### PR TITLE
Translation of page-weight chapter to Japanese

### PR DESCRIPTION
**Pull request by ksakae1216**
Please review for me.
Part of [#586](https://github.com/HTTPArchive/almanac.httparchive.org/issues/586)

### CODE REVIEW

File: src/content/ja/2022/page-weight.md [hunk 1]
```diff
@@ -0,0 +1,446 @@
...
+#TODO - Review and update chapter description
+description: 2022年のWeb Almanacのページの重さの章は、ページの重さが重要である理由、帯域幅、複雑なページ、時間と共に変化するページの重さ、ページリクエスト、ファイル形式について説明しています。
+authors: [fellowhuman1101, dwsmart]
+reviewers: [CSteele-gh]
+analysts: [drohe]
+editors: []
+translators: [ksakae1216]
+results: https://docs.google.com/spreadsheets/d/1JvJMiRsL6T9m_NEBHFh-rrQmU5a-ufdOKriSJbrEN8M/
+fellowhuman1101_bio: Jamie Indigoはロボットではありませんが、ボットの言葉を話します。技術的なSEOとして<a hreflang="en" href="https://www.deepcrawl.com">DeepCrawl</a>で、彼らは検索エンジンがウェブをどのようにクロール、レンダリング、そしてインデックス化するかを研究しています。彼らは野生のJavaScriptを手懐けることと、レンダリング戦略の最適化が大好きです。仕事をしていないとき、Jamieはホラー映画やグラフィックノベル、ダンジョンズ＆ドラゴンズが好きです。
+dwsmart_bio: Dave Smart is a developer and technical search engine consultant at <a hreflang="en" href="https://tamethebots.com">Tame the Bots</a>. They love building tools and experimenting with the modern web, and can often be found at the front in a gig or two.
```
Comment (tunetheweb) [inline src/content/ja/2022/page-weight.md:13]: Can we translate this?

File: src/content/ja/2022/page-weight.md [hunk 2]
```diff
@@ -0,0 +1,446 @@
+---
+#See https://github.com/HTTPArchive/almanac.httparchive.org/wiki/Authors'-Guide#metadata-to-add-at-the-top-of-your-chapters
+title: ページの重さ
+#TODO - Review and update chapter description
+description: 2022年のWeb Almanacのページの重さの章は、ページの重さが重要である理由、帯域幅、複雑なページ、時間と共に変化するページの重さ、ページリクエスト、ファイル形式について説明しています。
+authors: [fellowhuman1101, dwsmart]
+reviewers: [CSteele-gh]
+analysts: [drohe]
+editors: []
```
Comment (tunetheweb) [inline src/content/ja/2022/page-weight.md:9]: Could you add me as an editor (also in other langs if you don't mind) as I did edit this in #3159

**Review by tunetheweb — CHANGES_REQUESTED** [review]
LGTM

File: src/content/ja/2022/page-weight.md (hunk 1 above)
Comment (ksakae1216) [reply src/content/ja/2022/page-weight.md]: @tunetheweb Thanks, I translated

**Review by tunetheweb — APPROVED** [review]

**Pull request merged**